src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -0,0 +1,47 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@NoArgsConstructor
+@Getter
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue
+    private Long orderId;
+
+    @OneToMany(cascade = CascadeType.ALL)
+    @JoinTable(
+            name = "join_orders_carts",
+            joinColumns = {@JoinColumn(name = "order_id")},
+            inverseJoinColumns = {@JoinColumn(name = "cart_id")})
+    private List<Cart> carts = new ArrayList<>();
+
+    @Column(name = "orderDate")
+    private LocalDateTime orderDate;
+
+    @Column(name = "orderPaymentDate")
+    private LocalDateTime orderPaymentDate;
+
+    @Column(name = "orderValue")
+    private int orderValue;
+
+    @Column(name = "orderAmountPaid")
+    private int orderAmountPaid;
+
+    public Order(List<Cart> carts, LocalDateTime orderDate, LocalDateTime orderPaymentDate, int orderValue, int orderAmountPaid) {
```

Comment:
ja jestem za tym, żeby nie komplikować i przyjąć założenie, że płacimy za zamówienie w momencie jego składania. wtedy nie są potrzebne pola
orderPaymentDate
orderAmountPaid

Reply:
orderValue może być z kolei wyliczana metodą, a nie wpisywana z palca przy tworzeniu obiektu

Reply:
+ 1:1 relacja między koszykiem, a zamówieniem

Reply:
Mi się też wydaje że jak zrobimy relacje 1:1 koszyk->order to będzie OK, zawsze można dodać dodatkowe opcje jak będzie wszystko działać